src/environment.js

```diff
@@ -147,8 +152,7 @@ var Environment = Obj.extend({
                 }
 
                 // Resolve name relative to parentName
-                if (parentName && (name.indexOf("./") == 0 ||
-                                   name.indexOf("../") == 0)) {
```

Comment:
That makes sense to me. I like that better than forcing all loaders to implement this same logic if they want to copy the default behavior. If you want to do what @boutell wants, you just always return `true` from that method.